components/ui/themes/PurpleTheme.ts

```diff
@@ -1581,7 +1577,7 @@ export const PurpleTheme = {
     // https://ethercreative.github.io/react-native-shadow-generator/
 
     if (level === 0) {
-      return null;
+      return {};
```

Comment:
what's the difference null vs empty object

Reply:
When we are using the elevation style, it is expecting an object not a null value. For the testing the typescript was showing an error for that. So I have changed to {}